lib/rules/aria-deprecated-role.json

```
@@ -0,0 +1,15 @@
+{
+  "id": "aria-deprecated-role",
+  "selector": "[role]",
+  "matches": "no-empty-role-matches",
+  "tags": ["cat.aria", "wcag2a", "wcag412", "EN-301-549", "EN-9.4.1.2"],
+  "impact": "minor",
+  "actIds": ["674b10"],
+  "metadata": {
+    "description": "Ensures elements do not use deprecated roles",
+    "help": "ARIA roles used must not be deprecated"
```

I think you mean this:

```suggestion
    "help": "Deprecated ARIA roles must not be used"
```